tests/integration_tests/performance/test_boottime.py

```diff
@@ -22,8 +22,17 @@
 TIMESTAMP_LOG_REGEX = r'Guest-boot-time\s+\=\s+(\d+)\s+us'
 
 
+def test_no_boottime(test_microvm_with_api):
+    """Check boottimer no present by default."""
+    _ = _configure_vm(test_microvm_with_api)
```

Comment:
nit: I would rename this function for clarity - name is misleading, I had to look it up to make sure it starts/runs the VM:
```suggestion
    _ = _configure_and_run_vm(test_microvm_with_api)
```

Reply:
Good idea, I was a little confused about this, too. I will rename it.